modules/sendgrid/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "zone_name" {
+  description = "route53 zone for CNAME records to be created"
+  type = string
+}
+
+variable "sendgrid_api_key_secret_name" {
+  description = "api_key for sendgrid api, used to authenticate domain once route53 records are created"
```

Comment:
How about something like "name of the secret in AWS secret manager that contains the sendgrid API key"? Also could you please run the pre-commit hook so this stuff gets populated into the readme?